### DISCUSSION TITLE
Name our penguin! :penguin:

### DISCUSSION THREAD

**Discussion by troglobit** [kernelkit/infix#172] [Polls]
We have many great ideas for how to show off our fancy new [Infix](https://github.com/kernelkit/infix) logo.  Pins, decals, t-shirts, and more of course :smiley: 

![101700219](https://github.com/kernelkit/infix/assets/183517/4383b165-008b-4bd5-ad02-d6dbcdf16292)

However, when we drafted it we never really got to name our RJ45 penguin.  Our *"ghost in the Ethernet port"*.  Some of us have affectionately started calling him Blinky, since he lives in a slot of a product which is most happy when it blinks.  Also as an homage to Blinky the ghost that Pac-Man chases around in the classic game.

**Comment by wkz** [comment]
--R2-D2-- R4-J5?

**Comment by troglobit** [comment]
> --R2-D2-- R4-J5?

Not sure if anyone understands obscure movie references from the previous millennia. At least Pac-Man is a [timeless classic](https://gameishard.gg/news/the-timeless-appeal-of-pac-man/126801/) :wink:

**Reply by wkz** [reply]
I'm not opposed to "Binky", but to claim that it has more name recognition than "R2-D2" seems like a stretch :wink:

**Reply by troglobit** [reply]
I said what I said. Also Star Trek rulez! 🖖

**Comment by Stekartaro** [comment]
My suggestion is: **Finix**!
Infix => Finix ≈ Pheonix
A Pheonix is a mytical, majestic, bird-creature - who's appearance _some_ "wisemen" would say the Linux penguin also possess!
Also, a phoenix obtains new life by rising from the ashes of its predecessor: This could be applied to your IRL journey going from WeOS to Infix!

**Reply by troglobit** [reply]
Nice suggestion!  Granted I'm biased since I'm the author of [Finit](https://github.com/troglobit/finit/) :smile:

**Reply by firasuke** [reply]
Might as well throw in "Fuinix" or "Phuinix" to further include letters from the penguin.

**Comment by wzir-23** [comment]
Portos (variation of Porthos, Port + OS)?

**Reply by troglobit** [reply]
Heh, cool idea!  Also a hat tip to Star Trek again, Captain Archer's dog in Enterprise :smiley: :vulcan_salute:

**Comment by mattiaswal** [comment]
Pether is my suggestion, the Ethernet penguin!

**Comment by jovatn** [comment]
Perhaps Yax? From Yang and Tux.

**Comment by firasuke** [comment]
Pingu (The penguin show + the `ping` utility)

**Reply by troglobit** [reply]
This! I love Pingu (the show) ever since my kids were small, and ping in is like my favorite network troubleshooting tool.
Great suggestion :heart_eyes:

**Comment by jovatn** [comment]
Jack! (for RJ45 jack)

**Reply by troglobit** [reply]
Cool, I like it! Arne suggested Jack Gwyn on Linkedin: "Jack since it is a Jack and the Gwyn is taken from the Welsh ‘Pen Gwyn’ for Penguin." :smiley:

**Reply by troglobit** [reply]
We have a winner, @wkz (the creator) agrees that Jack/Jacky is the most suitable name :smiley::+1:
